modules/dxp/apps/search-experiences/search-experiences-service/src/main/resources/com/liferay/search/experiences/internal/model/listener/dependencies/rescore_by_text_embedding.json

```diff
@@ -90,7 +90,7 @@
 						},
 						{
 							"defaultValue": 1,
-							"helpText": "Only items above this score will be rescored.",
+							"helpText": "Sets the minimum score of items to be rescored.",
```

Comment:
Wordsmith Petteri's:
> Set the minimum score for a result to be rescored.

Alternative:
> Results with a lower score are not eligible for rescoring.

Reply:
👍  for the first one "Set the minimum score for a result to be rescored." since it describes what *will* happen.